components/mpas-ocean/cime_config/buildnml

```diff
@@ -295,6 +303,8 @@ def buildnml(case, caseroot, compname):
             ic_prefix = 'mpaso.IcoswISC30E3r5.rstFromG-chrysalis'
         if ocn_ismf == 'data':
             data_ismf_file = 'prescribed_ismf_adusumilli2020.IcoswISC30E3r5.20231120.nc'
+        if ocn_tidal_mixing == 'true':
+            u_tidal_rms_file = 'IcoswISC30E3r5_velocityTidalRMS_CATS2008.nc'
```

Comment:
Just a note that I have Compass generating files with a slightly different naming convention:
```
velocityTidalRMS_CATS2008.IcoswISC30E3r5.20231120.nc
```
This is how I ordered things for the data ice-shelf melt and data iceberg freshwater fluxes, and it's a good idea to have a date stamp.

It think it might be good rename the existing files with this convention or at least to add a date stamp to each of them.